docs/en/latest/FAQ.md

```diff
@@ -594,6 +594,36 @@ The differences between the two are described in the table below:
 | Used when there are property changes that needs to be propagated across all configuration instances of a Plugin. | Used when you need to reuse a common set of configuration instances so that it can be extracted to a `plugin-config` and bound to different Routes. |
 | Takes effect on all the entities bound to the configuration instances of the Plugin.                             | Takes effect on Routes bound to the `plugin-config`.                                                                                                |
 
+## After deploying Apache APISIX, how to detect the survival of the APISIX data plane?
+
+You can create a route named `health-info` and enable the [fault-injection](https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/fault-injection.md) plugin (where YOUR-TOKEN is the user's token; 127.0.0.1 is the IP address of the control plane, which can be modified by yourself):
```

Comment:
Please use the Website's URL instead of GitHub relative links for better SEO (internal links building)